docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+# Your RAG application is a communication system.
+
+Retrieval-Augmented Generation has emerged as a standard approach of implementing LLMs in large organizations. The grounding of LLM generation on existing data ensures an increased robustness to hallucinations, adaptation to in-domain knowledge and updatability. Yet despite proven achievements RAG is still commonly seen as a temporary workaround until the advent of models with an improved context length and more intelligent capabilities.
+
+After designing multiple RAG systems over the past year, I’ve come to believe that RAG signals a more radical transformation: the integration of LLMs in new forms of flexible expert systems. Effective RAG means ingesting even heterogeneous, badly formatted data sources and contextualizing new knowledge on the fly. RAG applications should aspire to become a communication system, taking the best features out of language models (adaptability, universal content translation) but also moderating their shortcomings (alignment mismatch, hallucinations, lack of explainability).
+
+As Yan et al. accurately [called it](https://applied-llms.org/#dont-forget-keyword-search-use-it-as-a-baseline-and-in-hybrid-search): “Models are likely to be the least durable component in the system”. They can be swapped, combined or disabled. What this means for the months and the years to come is that, rather than being replaced by some disruptive LLM technology like long context, RAG will eat LLMs. Embedding search is largely repurposed as an RAG technique. These days, I am mostly fine-tuning with RAG in mind. I recently co-founded PleIAs to get even one step further by pre-training a new generation of language models for RAG end use, and increased support for document processing use cases.
+
```

Comment:
"embedding search" = 1) searching for embeddings OR 2) incorporating search
?

Reply:
I believe "embedding search" refers to a search system based on vector embeddings.

I believe that sentence means something like: "Majority of vector search deployments today fall under the RAG technique. These days, I'm mostly fine-tuning language models with RAG in mind."

The slightly unclear part is whether the fine-tuning refers to the generative LLMs used in RAG (generation) or the embedding models also used in RAG (retrieval) - because both can be fine-tuned.